src/main/java/org/craftercms/studio/api/v1/exception/repository/InvalidRemoteRepositoryException.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Crafter Studio Web-content authoring solution
+ * Copyright (C) 2007-2018 Crafter Software Corporation.
```

Comment:
* Copyright (C) 2007-2018 Crafter Software Corporation.
needs to be
 * Copyright (C) 2007-2018 Crafter Software Corporation. All rights reserved.

Please update all files in this PR.